examples/xml/basic/tallies.xml

```diff
@@ -6,7 +6,6 @@
   </filter>
 
   <filter id="2" type="energy">
-    <bins>0 20.0e6</bins>
```

Comment:
Looks like this change accidentally slipped in

Reply:
Oops, just fixed that!